storage/connect/CMakeLists.txt

```diff
@@ -251,12 +278,16 @@ IF(CONNECT_WITH_JDBC)
     Mongo2Interface.java Mongo3Interface.java
     JavaWrappers.jar)
     add_definitions(-DJAVA_SUPPORT)
+    add_feature_info(CONNECT_JDBC "ON" "Support for JDBC in the CONNECT storage engine")
     IF(CONNECT_WITH_MONGO)
       add_definitions(-DMONGO_SUPPORT)
     ENDIF()
   ELSE()
     SET(JDBC_LIBRARY "")
+    add_feature_info(CONNECT_JDBC "OFF" "Support for JDBC in the CONNECT storage engine")
   ENDIF()
+ELSE(CONNECT_WITH_JDBC)
+  add_feature_info(CONNECT_JDBC "OFF" "Support for JDBC in the CONNECT storage engine")
```

Comment:
3x  add_feature_info(CONNECT_JDBC ...)
Would it be possible to simplify andf reduce it?